CMakeLists.txt

```diff
@@ -62,6 +62,7 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 find_package(YCM REQUIRED)
 find_package(YARP 3.3)
 find_package(iDynTree 3.0)
+find_package(Eigen3)
```

Comment:
Could it make sense to add `REQUIRED` to all?

Reply:
Right

Reply:
Done in https://github.com/robotology/idyntree-yarp-tools/pull/4/commits/1c084158fc6e0c909f2a33434057b195d8d4d0ac